constructors/aarch64/construct.yaml

```diff
@@ -0,0 +1,20 @@
+name: conda_bootstrap
+version: 1.0.0
+
+channels:
+  - https://conda.anaconda.org/c4aarch64
+  - https://conda.anaconda.org/jjhelmus/label/aarch64_bootstrap
+
+conda_default_channels:
+  - https://conda.anaconda.org/c4aarch64
+  - https://conda.anaconda.org/jjhelmus/label/aarch64_bootstrap
```

Comment:
I would advise against the version of conda in aarch64_bootstrap. I had alot of trouble adding and removing channels with it.